QRCoder/QRCodeGenerator.cs

```diff
@@ -1015,19 +1015,7 @@ private static int GetCountIndicatorLength(int version, EncodingMode encMode)
 
         private static int GetDataLength(EncodingMode encoding, string plainText, string codedText, bool forceUtf8)
         {
-            return forceUtf8 || IsUtf8(encoding, plainText, forceUtf8) ? (codedText.Length / 8) : plainText.Length;
-        }
-
-        private static bool IsUtf8(EncodingMode encoding, string plainText, bool forceUtf8)
-        {
-            return (encoding == EncodingMode.Byte && (!IsValidISO(plainText) || forceUtf8));
-        }
-
-        private static bool IsValidISO(string input)
-        {
-            var bytes = Encoding.GetEncoding("ISO-8859-1").GetBytes(input);
-            var result = Encoding.GetEncoding("ISO-8859-1").GetString(bytes);
-            return String.Equals(input, result);
+            return encoding == EncodingMode.Byte ? (codedText.Length / 8) : plainText.Length;
```

Comment:
```suggestion
            return encoding == EncodingMode.Byte ? (int)((uint)codedText.Length / 8) : plainText.Length;
```
do we care about such things here too?

Reply:
Yes; we certainly can optimize this PR also.

I need to come back to this PR; but for now I was leaving it alone until I'm finished with performance PRs.